resource/resource.h

```diff
@@ -1,124 +1,124 @@
-#define IDC_BUTTON4                     1032
-#define IDC_MINSTACK_ADD                1032
-#define IDC_LIST1                       1034
-#define IDC_MINSTACK_LIST               1034
-#define IDC_HOLD_LIST                   1036
-#define IDC_BUTTON2                     1046
-#define IDC_ROUTE_COPY                  1046
-#define IDC_SRD_ROUTE_COPY              1046
-#define IDC_HOLD_CLOSE                  1046
-#define TIMER1_EDIT_DURATION            1047
-#define TIMER2_EDIT_DURATION            1048
-#define IDC_TIME_FORMAT_CHECK           1048
-#define TIMER3_EDIT_DURATION            1049
-#define GS_TIME_FORMAT_CHECK            1049
-#define TIMER4_EDIT_DURATION            1050
-#define TIMER5_EDIT_VALUE               1051
-#define TIMER5_EDIT_DURATION            1051
-#define TIMER1_LABEL                    1052
-#define TIMER_CONFIG_ENABLED_LABEL      1053
-#define TIMER_CONFIG_DURATION_LABEL     1054
-#define TIMER2_LABEL                    1055
-#define TIMER3_LABEL                    1056
-#define TIMER4_LABEL                    1057
-#define TIMER5_LABEL                    1058
-#define IDC_MINSTACK_DELETE             1060
-#define IDC_MINSTACK_DOWN               1061
-#define IDC_MINSTACK_UP                 1062
-#define IDC_COMBO2                      1064
-#define IDC_MINSTACK_SELECT             1064
-#define IDC_MINSTACK_SELECTED_LABEL     1065
-#define IDC_RPS_DISPLAY_CHECK           1066
-#define IDC_RPS_SELECTED_LABEL          1067
-#define IDC_RPS_LIST                    1068
-#define IDC_RPS_UP                      1069
-#define IDC_RPS_DOWN                    1070
-#define IDC_RPS_DELETE                  1071
-#define IDC_RPS_SELECT                  1072
-#define IDC_RPS_ADD                     1073
-#define IDC_SRD_ORIGIN_STATIC           1075
-#define IDC_SRD_DESTINATION             1076
-#define IDC_SRD_CRUISE                  1078
-#define IDC_STATIC_REQUIRED             1079
-#define IDC_SRD_CRUISE_STATIC           1080
-#define IDC_SRD_DESTINATION_STATIC      1081
-#define IDC_SRD_RESULTS                 1085
-#define IDC_BUTTON1                     1086
-#define IDC_SRD_SEARCH                  1086
-#define IDC_HOLDS_SAVE                  1086
-#define IDC_EDIT1                       1087
-#define IDC_SRD_NOTES                   1087
-#define IDC_NOTES_STATIC                1088
-#define IDC_HOLD_PARAMS_MIN_STATIC      1089
-#define IDC_HOLD_PARAMS_MAX_STATIC      1090
-#define IDC_HOLD_MIN_SPIN               1091
-#define IDC_HOLD_MAX                    1092
-#define IDC_HOLD_MAXIMUM                1092
-#define IDC_SPIN2                       1093
-#define IDC_HOLD_MAX_SPIN               1093
-
-// Next default values for new objects
-// 
-#ifdef APSTUDIO_INVOKED
-#ifndef APSTUDIO_READONLY_SYMBOLS
-#define _APS_NEXT_RESOURCE_VALUE        121
-#define _APS_NEXT_COMMAND_VALUE         40001
-#define _APS_NEXT_CONTROL_VALUE         1094
-#define _APS_NEXT_SYMED_VALUE           101
-#endif
-#endif
+﻿//{{NO_DEPENDENCIES}}
```

Comment:
I don't see what this changeset is? 👁️

Reply:
As the branch is old I think this is mostly Git complaining about newlines